spec/lib/kimono_labs_client/web_client_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+require 'support/vcr'
+
+RSpec.describe KimonoLabsClient::WebClient do
+  context 'initializing the web client' do
+    it 'raises an error without an api key' do
+      expect { described_class.new }.to raise_error
```

Comment:
What kind of error? It might be good to be more specific here as to what exception class is being raised.